<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>System-ui</family>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">Heu-Fütterung</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label_rgv_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>581</width>
     <height>131</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>4000</width>
     <height>1200</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>70</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="b_heu_fuetterung">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>1280</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>35</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="text">
    <string>Fütterung</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_settings">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>20</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../Futterwagen/Futterkarre/icons/icons.qrc">
     <normaloff>:/icons/Gear.png</normaloff>
     <normalon>:/icons/Gear.png</normalon>:/icons/Gear.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../Futterwagen/Futterkarre/icons/icons.qrc">
     <normaloff>:/icons/Back.png</normaloff>:/icons/Back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next_rgv">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>140</y>
     <width>150</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../Futterwagen/Futterkarre/icons/icons.qrc">
     <normaloff>:/icons/Next.png</normaloff>:/icons/Next.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>680</y>
     <width>1200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>580</y>
     <width>1200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="txt_rohprotein">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>600</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rohprotein:</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_h_rohfaser">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>600</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rohfaser:</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_h_fruktan">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>600</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fruktan:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_h_reload">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>140</y>
     <width>150</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../Futterwagen/Futterkarre/icons/icons.qrc">
     <normaloff>:/icons/Reload.png</normaloff>:/icons/Reload.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_h_rohprotein">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>630</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>300</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic</family>
     <pointsize>32</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_h_rohfaser">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>630</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>300</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic</family>
     <pointsize>32</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_h_fruktan">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>630</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>300</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic [urw]</family>
     <pointsize>32</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="txt_kg_2">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>490</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic</family>
     <pointsize>50</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>kg</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fu_entnommen">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>330</y>
     <width>681</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic</family>
     <pointsize>200</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="styleSheet">
    <string>background-color: transparent;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_box">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>300</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BOX</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>600</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Sans</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_karre_gewicht_anzeigen">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>630</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>300</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic</family>
     <pointsize>32</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_box_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>600</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rest</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_h_extra">
   <property name="geometry">
    <rect>
     <x>629</x>
     <y>140</y>
     <width>191</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>34</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>EXTRA</string>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_rgv_alter_jahre">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>211</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>300</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rgv_gewicht_kg">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>241</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1501</width>
     <height>401</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>b_heu_fuetterung</zorder>
  <zorder>label_rgv_name</zorder>
  <zorder>btn_settings</zorder>
  <zorder>btn_back</zorder>
  <zorder>btn_next_rgv</zorder>
  <zorder>line_2</zorder>
  <zorder>line_1</zorder>
  <zorder>txt_rohprotein</zorder>
  <zorder>txt_h_rohfaser</zorder>
  <zorder>txt_h_fruktan</zorder>
  <zorder>btn_h_reload</zorder>
  <zorder>label_h_rohprotein</zorder>
  <zorder>label_h_rohfaser</zorder>
  <zorder>label_h_fruktan</zorder>
  <zorder>txt_kg_2</zorder>
  <zorder>label_fu_entnommen</zorder>
  <zorder>label_box</zorder>
  <zorder>exit</zorder>
  <zorder>label_karre_gewicht_anzeigen</zorder>
  <zorder>label_box_2</zorder>
  <zorder>btn_h_extra</zorder>
  <zorder>label_rgv_alter_jahre</zorder>
  <zorder>label_rgv_gewicht_kg</zorder>
 </widget>
 <resources>
  <include location="../../Futterwagen/Futterkarre/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
